docs/DESIGN.md

```diff
@@ -90,12 +90,58 @@ statement can be optimized to a jump table.
 A manager remote endpoint allows users to interact with the `EventManger`
 (as a `SubscriberOps` trait object) from a different thread of execution.
 This is particularly useful when the `EventManager` owns the subscriber object
-that the user wants to interact with, but the `EventManager` being on a
-different thread requires synchronized handles.
+the user wants to interact with, and the communication happens from a separate
+thread. This functionality is gated behind the `remote_endpoint` feature.
 
 The current implementation relies on passing boxed closures to the manager and
 getting back a boxed result. The manager is notified about incoming invocation
 requests via an `EventFd` which is added by the manager to its internal run
```

Comment:
```suggestion
requests via an [`EventFd`](https://docs.rs/vmm-sys-util/latest/vmm_sys_util/eventfd/struct.EventFd.html) which is added by the manager to its internal run
```

Reply:
I prefer adding it all the time but I didn't specifically search for occurrences. This one popped into view because I was reading the doc more carefully while running the sample code.